src/Integrations/Integrations/SQLSRV/SQLSRVIntegration.php

```diff
@@ -0,0 +1,256 @@
+     */
+    public function getName()
+    {
+        return self::NAME;
+    }
+
+    /**
+     * Load the integration
+     *
+     * @return int
+     */
+    public function init()
+    {
+        if (!extension_loaded('sqlsrv')) {
+            return Integration::NOT_AVAILABLE;
+        }
+
+        $integration = $this;
+
+        // sqlsrv_connect ( string $serverName [, array $connectionInfo] ) : resource
+        \DDTrace\trace_function('sqlsrv_connect', function (SpanData $span, $args, $retval) use ($integration) {
+            $connectionMetadata = $integration->extractConnectionMetadata($args);
+            ObjectKVStore::put($this, SQLSRVIntegration::CONNECTION_TAGS_KEY, $connectionMetadata);
+            self::setDefaultAttributes($connectionMetadata, $span, 'sqlsrv_connect');
+
+            $integration->detectError($retval, $span);
+        });
+
+        if (PHP_MAJOR_VERSION > 5) {
+            // sqlsrv_query ( resource $conn , string $query [, array $params [, array $options ]] ) : resource
+            \DDTrace\install_hook('sqlsrv_query', function (HookData $hook) use ($integration) {
+                list(, $query) = $hook->args;
+
+                $span = $hook->span();
+                self::setDefaultAttributes($this, $span, 'sqlsrv_query', $query);
+                $integration->addTraceAnalyticsIfEnabled($span);
+
+                ObjectKVStore::put($this, SQLSRVIntegration::QUERY_TAGS_KEY, $query);
+
+                DatabaseIntegrationHelper::injectDatabaseIntegrationData($hook, 'sqlsrv', 1);
+            }, function (HookData $hook) use ($integration) {
+                $span = $hook->span();
+                if (is_object($hook->returned)) {
+                    ObjectKVStore::propagate($this, $hook->returned, SQLSRVIntegration::CONNECTION_TAGS_KEY);
+                }
+
+                $result = $hook->returned;
+
+                $this->setMetricNumRows($span, $result);
+
+                $integration->detectError($hook->returned, $span);
+            });
+
+            // sqlsrv_prepare ( resource $conn , string $query [, array $params [, array $options ]] ) : resource
+            \DDTrace\install_hook('sqlsrv_prepare', function (HookData $hook) use ($integration) {
+                list(, $query) = $hook->args;
+
+                ObjectKVStore::put($this, SQLSRVIntegration::QUERY_TAGS_KEY, $query);
+
+                $span = $hook->span();
+                self::setDefaultAttributes($this, $span, 'sqlsrv_prepare', $query);
+
+                DatabaseIntegrationHelper::injectDatabaseIntegrationData($hook, 'sqlsrv', 1);
+            }, function (HookData $hook) use ($integration) {
+                $span = $hook->span();
+                if (is_object($hook->returned)) {
+                    ObjectKVStore::propagate($this, $hook->returned, SQLSRVIntegration::CONNECTION_TAGS_KEY);
+                }
+
+                $integration->detectError($hook->returned, $span);
+            });
+        } else {
```

Comment:
In theory, as we're only in [maintenance mode for PHP5](https://docs.datadoghq.com/tracing/trace_collection/compatibility/php/#runtime-support-policy-for-php-apm), you didn't have to do this. But no worries if it is done now

Reply:
(It just has no effect on PHP 5 as it's not declared in the IntegrationsLoader for PHP 5)